File: mapir-android-service-sdk/src/main/java/ir/map/servicesdk/model/inner/DistanceMatrixPointRequest.java
package ir.map.servicesdk.model.inner;

public class DistanceMatrixPointRequest {

    private String id;
    private double latitude;
    private double longitude;

    public DistanceMatrixPointRequest(String id, double latitude, double longitude) {
        this.id = id;
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public DistanceMatrixPointRequest(int id, double latitude, double longitude) {
        this.id = String.valueOf(id);
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public String getId() {
        return id;
    }

    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }
}

File: mapir-android-service-sdk-ktx/src/main/java/ir/map/servicesdk/model/inner/DistanceMatrixPointRequest.kt
package ir.map.servicesdk.model.inner

class DistanceMatrixPointRequest {
    var id: String
    var latitude: Double
    var longitude: Double

    constructor(id: String, latitude: Double, longitude: Double) {
        this.id = id
        this.latitude = latitude
        this.longitude = longitude
    }

    constructor(id: Int, latitude: Double, longitude: Double) {
        this.id = id.toString()
        this.latitude = latitude
        this.longitude = longitude
    }
}